lib/classes/mapi-request.js

```diff
@@ -4,7 +4,7 @@ var parseToken = require('@mapbox/parse-mapbox-token');
 var xtend = require('xtend');
 var EventEmitter = require('eventemitter3');
 var urlUtils = require('../helpers/url-utils');
-
+var constants = require('../constants');
 var requestId = 1;
```

Comment:
I'd prefer to conventionally keep an empty line separating imports from the rest of the code.